tips/string-raw.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Non-escaped strings using String.raw"
+language: "javascript"
+---
+
+String.raw`C:\\Users\gbrlmngr\new folder`; 
+  // => "C:\\Users\gbrlmngr\new folder"
```

Comment:
What is this doing exactly? Looks to me it's returning the same value you're passing into `String.raw`

Reply:
Exactly! Using quotes or double quotes instead of `String.raw` would evaluate the escape characters (e.g. `\`, `\n`), so it would print:
```js
"C:\Usersgbrlmngr
ew folder"
```